Added badge notice for `Atom-Ink` and added new badge spec (#148)
pulsar-edit/package-backend: docs/reference/Admin_Actions.md

With that said this document will serve as the ongoing history of administrative actions that must be taken against the backend.

## 2023 - April 15

### Uber-Juno

> The [Uber-Juno](https://github.com/JunoLab/uber-juno) package uses an older unsupported coding style, sometimes called JavaScript Sloppy Mode, that's no longer supported in newer versions of NodeJS. Meaning the package crashes as soon as installed, and cannot be used without the user manually editing the source code.

Reviewer: as per the observation by comment https://github.com/pulsar-edit/pulsar/issues/482#issuecomment-1506542131 , it said in that error the following ...

> Modules are automatically in strict mode.

and since no issues are shown after that change, I think "it uses [Sloppy Mode](https://developer.mozilla.org/en-US/docs/Glossary/Sloppy_mode)" no longer applies...

> One of the Pulsar contributors has [submitted a fix](https://github.com/JunoLab/uber-juno/pull/85) for this issue, although it has not been updated on the Pulsar Package Registry.

Reviewer: ```suggestion
[A fix](https://github.com/JunoLab/uber-juno/pull/85) from one of the Pulsar contributors has been merged, although it has not been updated on the Pulsar Package Registry.
```

Owing to the mention of "was accepted" in the section for Hydrogen (L22 in the master version)

Additionally, Uber-Juno relies on Atom-Ink, which is also broken without any fixes, as mentioned below.

> Due to these issues of the package, while the `main` branch is fixed, but it relies on broken dependencies, that render's the package as a whole still broken, and in which case will receive a [`broken`](./badge-spec.md#broken) badge.

Reviewer: Atom-Ink is also fixed.. so, this reason of "broken due to Atom-Ink" will have to be removed, and whole paragraph will have to be reworded to mention that issue with downloading as i mentioned in my previous comment:

`Juno Error: Installing julia-client@0.12.6 failed` as shown in the image at https://github.com/pulsar-edit/pulsar/issues/482#issuecomment-1506577042

### Atom-Ink

The [Atom-Ink](https://github.com/JunoLab/atom-ink) package has many coding practices that are no longer valid in JavaScript/NodeJS, such as using reserved words, or using variables without declaring them first, causing the package to crash without manually being edited on Pulsar.

One of the Pulsar contributors has [submitted a fix](https://github.com/JunoLab/atom-ink/pull/289) for this bug, but after four months of inactivity from the maintainer of the package it seems unlikely this will be resolved.

The package is MIT licensed, so if any devs out there would like to pick up the reigns, and get this package functional as a fork, please feel free to do so, and you'd be encouraged to contact the Pulsar Admins about this, so we can update the badges on this package appropriately.

> It is for this reason it is **not** recommended to install the `atom-ink` package. It will not work, unless you are comfortable editing the source code of the package manually, or until a community member decides to maintain a fork of said package, which in that event, installing that package would be the recommendation here, rather than not installing.

Reviewer: This whole will have to be reworded as well... as the issue with using reserved words was with `uber-juno`, fix for undeclared variable use has been merged.

smth similar to instructions in line 26-28 (section hydrogen) `pulsar -p install ...` can work but since this is a dependency for uber-juno, i don't know what complications regarding its install that way will be...

 also, the upstream hasn't tagged a release _yet_. so, the command will have to target a commit for the time being... 

i have nudged upstream to tag it, but ...
